packages/exists/tests/existsModule.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { expect } from 'chai';
+import {ExistsModule} from '../src'
+
+describe('existsModule', () => {
+    it('shouldCreate', () => {
```

Comment:
this test feels redundant. there is no way this test will fail. the new operator either throws error or creates something which is not `null`.
it will might check that the constructor doesn't throw, but I don't know if it's important

Reply:
You're right, this was a test to check if the testing infrastructure of the modules work.
We should remove it.